src/Components/Ratings/Ratings.jsx

```diff
@@ -0,0 +1,5 @@
+export const Ratings = (props) => {
+    return (
```

Comment:
suggestion: destructure props in the function itself. like function Ratings({ text }) { ... }